instrumentation/runtime/internal/x/x.go

```diff
@@ -0,0 +1,53 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// Package x contains support for OTel SDK experimental features.
+//
+// This package should only be used for features defined in the specification.
+// It should not be used for experiments or new project ideas.
+package x // import "go.opentelemetry.io/otel/sdk/internal/x"
+
+import (
+	"os"
+	"strings"
+)
+
+// Resource is an experimental feature flag that defines if the deprecated
+// runtime metrics should be produced. During development of the new
+// conventions, it is enabled by default.
+//
+// To disable this feature set the OTEL_GO_X_DEPRECATED_RUNTIME_METRICS environment variable
+// to the case-insensitive string value of "false" (i.e. "False" and "FALSE"
+// will also enable this).
+var DeprecatedRuntimeMetrics = newFeature("DEPRECATED_RUNTIME_METRICS", true)
+
+// BoolFeature is an experimental feature control flag. It provides a uniform way
+// to interact with these feature flags and parse their values.
+type BoolFeature struct {
```

Comment:
This differs from the "Feature" pattern used elsewhere because I wanted to be able to have a default value of "true".  Rather than make the existing pattern more complex to support this, I simplified it to fit this packages' needs.